src/cloudsc_cuda/cloudsc/cloudsc_driver.cu

```diff
@@ -12,6 +12,7 @@
 
 #include <omp.h>
 #include "mycpu.h"
+// #include "dtype.h"
```

Comment:
This should probably be here but has likely caused problems because of duplicate definitions without the include guards?

Reply:
The include is within `cloudsc_driver.h` and  `cloudsc_driver.h` is included in  `cloudsc_driver.cu`